src/main/java/backend/resource/TurboLabel.java

```diff
@@ -211,4 +224,138 @@ public int hashCode() {
     public int compareTo(TurboLabel o) {
         return actualName.compareTo(o.getActualName());
     }
+
+    /**
+     * Returns the TurboLabel in labels that matches labelName
+     * Assumption: the labelName matches exactly 1 TurboLabel
+     *
+     * @param labels
+     * @param labelName
+     * @return
+     */
+    public static TurboLabel getMatchingTurboLabel(List<TurboLabel> labels, String labelName) {
+        assert labels.stream()
+                .filter(label -> label.getActualName().equals(labelName))
+                .findFirst()
+                .isPresent();
+        return labels.stream()
+                .filter(label -> label.getActualName().equals(labelName))
+                .findFirst()
+                .get();
```

Comment:
There is substantial repetition between these two statements. Any way to reduce that?